app/views/super_admin/allowlists/organisation_names/index.html.erb

```diff
@@ -15,8 +15,10 @@
         <%= link_to "Allow an organisation", new_super_admin_allowlist_organisation_name_path, class: "govuk-button", role: "button", draggable: "false", "data-module" => "govuk-button" %>
       </div>
     </div>
-    <%= label_tag "search_orgs_label", "Search for existing organisations on GovWifi before adding them", class: "govuk-label" %>
-    <%= select_tag "organisations_register", options_for_select(@fetched_organisations_from_register << "", ""), class: "govuk-select" %>
+    <%= label_tag "organisations_register", "Search for existing organisations on GovWifi before adding them", class: "govuk-label" %>
+    <%= select_tag "organisations_register", options_for_select(@fetched_organisations_from_register, ""), class: "govuk-select", id: "organisations_register" %>
```

Comment:
do we not need the `<< ""` bit in `options_for_select(@fetched_organisations_from_register << "", "")` anymore?